config/config.go

```diff
@@ -578,8 +582,9 @@ func DefaultRPCConfig() *RPCConfig {
 		MaxBodyBytes:   int64(1000000), // 1MB
 		MaxHeaderBytes: 1 << 20,        // same as the net/http default
 
-		TLSCertFile: "",
-		TLSKeyFile:  "",
+		TLSCertFile:          "",
+		TLSKeyFile:           "",
+		DiscardABCIResponses: true,
```

Comment:
Do we really want the default to be true here? We had discussed making new settings opt-in for the operator, and this one definitely comes with some tradeoffs.